clients/src/main/resources/common/message/UpdateFeaturesResponse.json

```diff
@@ -26,13 +26,13 @@
       "about": "The top-level error code, or `0` if there was no top-level error." },
     { "name": "ErrorMessage", "type": "string", "versions": "0+", "nullableVersions": "0+",
       "about": "The top-level error message, or `null` if there was no top-level error." },
-    { "name": "Results", "type": "[]UpdatableFeatureResult", "versions": "0+",
+    { "name": "Results", "type": "[]UpdatableFeatureResult", "versions": "0-1", "ignorable": true,
       "about": "Results for each feature update.", "fields": [
-      { "name": "Feature", "type": "string", "versions": "0+", "mapKey": true,
+      { "name": "Feature", "type": "string", "versions": "0-1", "mapKey": true,
```

Comment:
Do we need to cap the version in the sub fields? For example, DeleteTopicsRequest doesn't do that in v6.

Reply:
I did it to be safe, but I can remove.